x/stake/client/cli/query.go

```diff
@@ -139,7 +140,10 @@ func GetCmdQueryDelegation(storeName string, cdc *wire.Codec) *cobra.Command {
 			}
 
 			// parse out the delegation
-			delegation := types.MustUnmarshalDelegation(cdc, key, res)
+			delegation, err := types.UnmarshalDelegation(cdc, key, res)
+			if err != nil {
+				return errors.Errorf("cannot unmarshal delegation: %v", err)
```

Comment:
Wouldn't it be better to add a new error function in `types/errors.go` and use that instead ? you can use the `CodeInternal` as the `CodeType`